fili-core/src/main/resources/sampleData.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type" : "record",
+  "name" : "TUPLE_0",
+  "fields" : [ {
+    "name" : "FOO_ID",
+    "type" : [ "null", "int" ]
+  }, {
+    "name" : "FOO_DESC",
```

Comment:
This indentation looks wrong, since it aligns with 'fields' but is actually _inside_ the list that is the value of fields.